database/spanner/spanner_test.go

```diff
@@ -58,69 +55,99 @@ func TestMigrate(t *testing.T) {
+			name: "single statement, multi line, with semicolon, no comment",
 			multiStatement: `CREATE TABLE table_name (
-	id STRING(255) NOT NULL,
-) PRIMARY KEY (id);`,
+			id STRING(255) NOT NULL,
+		) PRIMARY KEY (id);`,
+			expected: []string{"CREATE TABLE table_name (\n  id STRING(255) NOT NULL,\n) PRIMARY KEY(id)"},
+		},
+		{
+			name: "multi statement, with trailing semicolon. no comment",
+			// From https://github.com/mattes/migrate/pull/281
+			multiStatement: `CREATE TABLE table_name (
+			id STRING(255) NOT NULL,
+		) PRIMARY KEY(id);
+
+		CREATE INDEX table_name_id_idx ON table_name (id);`,
 			expected: []string{`CREATE TABLE table_name (
-	id STRING(255) NOT NULL,
-) PRIMARY KEY (id)`},
+  id STRING(255) NOT NULL,
+) PRIMARY KEY(id)`, "CREATE INDEX table_name_id_idx ON table_name(id)"},
 		},
 		{
-			name: "multi statement, with trailing semicolon",
+			name: "multi statement, no trailing semicolon, no comment",
 			// From https://github.com/mattes/migrate/pull/281
 			multiStatement: `CREATE TABLE table_name (
-	id STRING(255) NOT NULL,
-) PRIMARY KEY(id);
+			id STRING(255) NOT NULL,
+		) PRIMARY KEY(id);
 
-CREATE INDEX table_name_id_idx ON table_name (id);`,
+		CREATE INDEX table_name_id_idx ON table_name (id)`,
 			expected: []string{`CREATE TABLE table_name (
-	id STRING(255) NOT NULL,
-) PRIMARY KEY(id)`, "CREATE INDEX table_name_id_idx ON table_name (id)"},
+  id STRING(255) NOT NULL,
+) PRIMARY KEY(id)`, "CREATE INDEX table_name_id_idx ON table_name(id)"},
 		},
 		{
-			name: "multi statement, no trailing semicolon",
+			name: "multi statement, no trailing semicolon, standalone comment",
 			// From https://github.com/mattes/migrate/pull/281
 			multiStatement: `CREATE TABLE table_name (
-	id STRING(255) NOT NULL,
-) PRIMARY KEY(id);
+			-- standalone comment
+			id STRING(255) NOT NULL,
+		) PRIMARY KEY(id);
 
-CREATE INDEX table_name_id_idx ON table_name (id)`,
+		CREATE INDEX table_name_id_idx ON table_name (id)`,
 			expected: []string{`CREATE TABLE table_name (
-	id STRING(255) NOT NULL,
-) PRIMARY KEY(id)`, "CREATE INDEX table_name_id_idx ON table_name (id)"},
+  id STRING(255) NOT NULL,
+) PRIMARY KEY(id)`, "CREATE INDEX table_name_id_idx ON table_name(id)"},
+		},
+		{
+			name: "multi statement, no trailing semicolon, inline comment",
+			// From https://github.com/mattes/migrate/pull/281
+			multiStatement: `CREATE TABLE table_name (
+			id STRING(255) NOT NULL, -- inline comment
+		) PRIMARY KEY(id);
+
+		CREATE INDEX table_name_id_idx ON table_name (id)`,
+			expected: []string{`CREATE TABLE table_name (
+  id STRING(255) NOT NULL,
+) PRIMARY KEY(id)`, "CREATE INDEX table_name_id_idx ON table_name(id)"},
 		},
 	}
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			if stmts := migrationStatements([]byte(tc.multiStatement)); !assert.Equal(t, stmts, tc.expected) {
+			stmts, err := cleanStatements([]byte(tc.multiStatement))
+			if err != nil {
+				t.Error()
```

Comment:
Make this `t.Fatal("Error cleaning statements:", err)`

Reply:
Fixed. Since we are already using testify I switched the whole if statement to use `require.NoError` which does the same but avoids the extra `if`. Let me know if this is an issue

Reply:
Cool, didn't know about `require.NoError()`!